src/ApplyExternalSettings.cpp

```diff
@@ -83,11 +82,11 @@ EmptyMassProcessTimer()
   static Validity last_empty_mass_validity;
   const NMEAInfo &basic = CommonInterface::Basic();
   PolarSettings &polar(CommonInterface::SetComputerSettings().polar);
-  Plane &plane( CommonInterface::SetComputerSettings().plane );
   if( basic.settings.empty_mass_available.Modified( last_empty_mass_validity ) ){
+    Plane &plane( CommonInterface::SetComputerSettings().plane );
     polar.glide_polar_task.SetEmptyMass(basic.settings.empty_mass);
     plane.empty_mass = basic.settings.empty_mass;
-    LogFormat("External Settings new empty mass %f", basic.settings.empty_mass );
+    PlaneGlue::WriteFile( plane );
```

Comment:
Should we really rewrite the whole plane file every time the vario decides to send a new value? What if a different vario repeats the (unmodified) value every second?

Reply:
to be discussed in the next PR, plane masses are removed for now